### DISCUSSION TITLE
how about Theia send canonical paths to the adapter

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-theia/theia#13875] [Q&A]
Hi @msujew 
    When Debug between theia and cdt-gdb-adapter, there is a problem with the path theia sends to the adapter, it is not only about cdt-gdb-adapter, maybe another adapter has the same problem, the discussion detail is [here](https://github.com/eclipse-cdt-cloud/cdt-cloud/discussions/42#discussioncomment-9940353), Do I need to raise an issue about the canonical paths problem?

**Comment by msujew** [comment]
Hey @wss29,

yes, please raise an issue for this.
